--- a/src/gn/desc_builder.cc
+++ b/src/gn/desc_builder.cc
@@ -165,16 +165,16 @@
   template <typename T>
   base::Value ToBaseValue(const std::vector<T>& vector) {
     base::ListValue res;
     for (const auto& v : vector)
       res.GetList().emplace_back(ToBaseValue(v));
-    return std::move(res);
+    return res;
   }
 
   base::Value ToBaseValue(const Scope* scope) {
     base::DictionaryValue res;
     Scope::KeyValueMap map;
     scope->GetCurrentScopeValues(&map);
     for (const auto& v : map)
       res.SetKey(v.first, ToBaseValue(v.second));
-    return std::move(res);
+    return res;
   }
